cfme/tests/configure/test_timeprofile.py

```diff
@@ -1,85 +1,113 @@
 # -*- coding: utf-8 -*-
 import fauxfactory
-import cfme.configure.settings as st
 import pytest
-from cfme.utils import error
-from cfme.utils.update import update
-from cfme.utils import version
+
+from cfme.configure.settings import TimeProfileAddFormView
 from cfme import test_requirements
-from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.utils.update import update
 
 
 pytestmark = [pytest.mark.tier(3),
               test_requirements.settings]
 
 
-def new_timeprofile():
-    return st.Timeprofile(description='time_profile' + fauxfactory.gen_alphanumeric(),
-                          scope='Current User',
-                          days=True,
-                          hours=True,
-                          timezone="(GMT-10:00) Hawaii")
-
-
 @pytest.mark.sauce
-def test_timeprofile_crud():
-    timeprofile = new_timeprofile()
-    timeprofile.create()
-    with update(timeprofile):
-        timeprofile.scope = 'All Users'
-    copied_timeprofile = timeprofile.copy()
-    copied_timeprofile.delete()
-    timeprofile.delete()
+def test_time_profile_crud(appliance):
+    """
+        This test case performs the CRUD operation.
+    """
+    collection = appliance.collections.time_profiles
+    time_profile = collection.create(
+        description='time_profile {}'.format(fauxfactory.gen_alphanumeric()),
+        scope='Current User',
+        days=True, hours=True,
+        timezone="(GMT-10:00) Hawaii")
+    with update(time_profile):
```

Comment:
I would like to see more assertion in this test.
Navigate to details for the newly created entity, and assert the view is displayed.
Navigate again after scope has been changed, check the details widgets for the correct scope

Reply:
In time profile there is no conventional details page.